api/docs/developer-system-initialization.md

```diff
@@ -100,8 +100,9 @@ git secrets --scan -r /path/to/directory
 ## Initial Smoke Test
 Before doing any development, you must run the following from `/api`:
 ```Shell
-./gradlew compileGeneratedJava appengineRun
+./gradlew compileJava
 ```
-This will generate compiled Java and MapStruct files that are necessary
-for the app to compile. On Macs, this command will never progress beyond 96%.
-After verifying that it builds and runs, it can safely be killed with `ctrl+c`.
+Before compiling, this will also generate Java files from MapStruct and OpenAPI (Swagger) 
+definitions which the app requires to compile. Depending on your personal development environment,
+you may need to frequently run `./gradlew clean compileJava` to pick up MapStruct/OpenAPI changes, 
```

Comment:
IntelliJ needs an assist for my configuration - dunno about everyone else

Reply:
Yes - I always need to run `compileGeneratedJava` to regenerate swagger models (though I'm sure there is a way to do it from Intellij). I sometimes see issues with mapstruct generation when switching branches and run compileJava to fix, though once it's working and I'm changing a mapper, I usually don't need to rerun compileJava